Dockerfile

```diff
@@ -5,32 +5,40 @@ RUN apt-get update && apt-get install --no-install-recommends -y build-essential
     zlib1g-dev pkg-config
 
 # Install libturbo-jpeg 1.4.2
-ADD http://sourceforge.net/projects/libjpeg-turbo/files/1.4.2/libjpeg-turbo-official_1.4.2_amd64.deb/download /tmp/libjpeg-turbo-official_1.4.2_amd64.deb
-RUN cd /tmp && dpkg -i /tmp/libjpeg-turbo-official_1.4.2_amd64.deb && \
-    echo /opt/libjpeg-turbo/lib64 > /etc/ld.so.conf.d/libjpeg-turbo.conf && ldconfig
+RUN wget -q https://sourceforge.net/projects/libjpeg-turbo/files/1.4.2/libjpeg-turbo-official_1.4.2_amd64.deb/download -O /tmp/libjpeg-turbo-official_1.4.2_amd64.deb && \
```

Comment:
The ADD command was downloading the sources each time `docker build` was called. This RUN operation does the same and it's cached by its docker layer, no need to download it each time.